### PR TITLE
docs: add cloud public traces docs

### PR DESCRIPTION
**Pull request by marcosnils**
Signed-off-by: Marcos Lilljedahl <marcosnils@gmail.com>

fixes https://linear.app/dagger/issue/GTM-2212/create-documentation-for-public-traces

### CODE REVIEW

File: docs/current_docs/configuration/cloud.mdx [hunk 1]
```diff
@@ -203,6 +203,17 @@ dagger -m github.com/shykes/daggerverse/hello@v0.3.0 call hello
 
 Trigger your CI workflow by pushing a commit or opening a pull request.
 
+### Public traces
+
+Dagger Cloud automatically detects if the traces your pipelines produce are coming
```
Comment (mircubed) [inline docs/current_docs/configuration/cloud.mdx:208]: I'd change this section to:

Dagger Cloud automatically detects if the traces produced by your pipelines originate from a CI system and a public repository. When both conditions are met, Dagger Cloud allows public access to these traces without requiring an invitation to the organization.

To modify this default setting, navigate to the organization settings page. Under the `Configuration` tab, toggle the `Trace Repository Visibility` switch to control access to trace data as needed.

Comment (vikram-dagger) [reply docs/current_docs/configuration/cloud.mdx:208]: +1

File: docs/current_docs/configuration/cloud.mdx [hunk 2]
```diff
@@ -203,6 +203,17 @@ dagger -m github.com/shykes/daggerverse/hello@v0.3.0 call hello
...
+from CI and a public repository. If so, Dagger Cloud will allow accessing such traces
+without the need to be invited to the organization.
+
+In case the above default behavior needs to be changed, you can do so from the organization
+settings page by toggling the `Trace Repository Visibility` switch under the `Configuration` tab.
+
+![public traces](/img/current_docs/api/public-traces.png)
```
Comment (vikram-dagger) [inline docs/current_docs/configuration/cloud.mdx:215]: We are highlighting sections in images with a red border 6px box. See https://docs.dagger.io/assets/images/manage-volumes-3188bad19829c3dd938d5a2607a0fdea.png for an example. Can you please update the image accordingly? Thanks.

Comment (mircubed) [reply docs/current_docs/configuration/cloud.mdx:215]: Screenshot 2024-11-06 at 11 29 50 AM


Here is the image to use

**Pull request merged**